src/devices/RotaryEncoder/QuadratureRotaryEncoder.cs

```diff
@@ -65,7 +75,10 @@ public QuadratureRotaryEncoder(GpioController controller, int pinA, int pinB, Pi
         /// <param name="pinA">Pin A that is connected to the rotary encoder. Sometimes called clk</param>
         /// <param name="pinB">Pin B that is connected to the rotary encoder. Sometimes called data</param>
         /// <param name="pulsesPerRotation">The number of pulses to be received for every full rotation of the encoder.</param>
-        public QuadratureRotaryEncoder(int pinA, int pinB, int pulsesPerRotation) : this(new GpioController(), pinA, pinB, PinEventTypes.Falling, pulsesPerRotation) { }
+        public QuadratureRotaryEncoder(int pinA, int pinB, int pulsesPerRotation)
```

Comment:
I'd recommend adding default args to other constructor and change order of args and remove this constructor completely